tests/fixtures.py

```diff
@@ -967,6 +979,25 @@ def snowflake_example_test_dataset_config(
     dataset_config.delete(db=db)
 
 
+@pytest.fixture
```

Comment:
We can make this a `session` level scope since the dataset won't change between tests